test/e2e/network/loadbalancer.go

```diff
@@ -1614,7 +1677,7 @@ var _ = common.SIGDescribe("LoadBalancers ESIPP [Slow]", func() {
 				return false, nil
 			}
 			ginkgo.By(fmt.Sprintf("Endpoint %v:%v%v returned client ip %v", ingressIP, svcTCPPort, path, clientIP))
-			if !strings.HasPrefix(clientIP, subnetPrefix[0]+".") {
+			if !subnetPrefix.Contains(net.ParseIP(clientIP)) {
```

Comment:
assert the clientIP is valid https://github.com/kubernetes/kubernetes/pull/114027/files#r1030216487

Reply:
Yup.